src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
+    private final Color[] colors = Color.values();
+
+    public static String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name().toLowerCase();
```

Comment:
Instead of calling 'Color.values()' multiple times, use the 'colors' array that you have already initialized. This will avoid creating a new array every time the method is called.